content/projects/understanding-loops/_index.md

```diff
@@ -65,6 +65,22 @@ Please note that one person can have multiple baskets. If you look at the data,
 
 ## Write some functionality
 
+### Important notes for Javascript solutions
+
+In Javascript it is best practice to use destructuring when passing arguments to a function. You can learn about destructuring here: [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment).
+
+Please make sure you use destructuring for all your functions. 
+
+```
+getCustomerBaskets(someJsonData, "sine@umuzi.org") // DON'T DO THIS
+getCustomerBaskets("sine@umuzi.org", someJsonData) // DON'T DO THIS EITHER
+
+// the problem with the above code is that if you get the order of the parameters wrong, then things break. 
+// your functions should work like this instead:
+
```

Comment:
I think here you are supposed to provide a code example of how the function should work?